packages/web/README.md

```diff
@@ -44,4 +49,39 @@ This code will make the transaction, let the user sign it using algosigner and s
 
 You can also use `web.sendTransaction()` or `web.signTransaction()` in a react app.
 
-**Note:** We don't support checkpoints yet. Currently `deployASA`, `deploySSC` functions don't work. User should directly pass assetIndex, appIndex instead of asaName, appName.
+### deployApp
+`deployer.deployApp` deploys stateful smart contract.
+
+Parameters:
+- approvalProgram:  approval program filename (must be present in assets folder)
+- clearProgram:  clear program filename (must be present in assets folder)
+- flags:  AppDeploymentFlags
+- payFlags:  Transaction Parameters
+- scTmplParams:  Smart contract template parameters (used only when compiling PyTEAL to TEAL). This is an optional parameter.
+- appName: name of the app to deploy. This name (if passed) will be used as
+the checkpoint "key", and app information will be stored agaisnt this name. This is an optional parameter.
+
+### Example
+```js
+// deployment
+const daoAppInfo = await deployer.deployApp(
+	"dao-app-approval.py",
+	"dao-app-clear.py",
+	{
+		sender: creator,
+		localInts: 9,
+		localBytes: 7,
+		globalInts: 4,
+		globalBytes: 2,
+		appArgs: appArgs,
+	},
+	{},
+	{},
+	"DAO App" // app name passed here
+);
+
+// now during querying, you only need this app name
+const appInfo = deployer.getApp("DAO App");
+```
+
+**Note:** We don't support checkpoints yet. Currently `deployASA`, `deployApp` functions don't work. User should directly pass assetIndex, appIndex instead of asaName, appName.
```

Comment:
Are we support deployASA and deployApp on web-mode? 
If we already, should we remove those line above.